amm/src/main/scala/ammonite/Main.scala

```diff
@@ -60,7 +60,7 @@ case class Main(predef: String = "",
     new Repl(
       inputStream, outputStream, errorStream,
       storage = storageBackend,
-      predef = augmentedPredef + "\n" + predef,
+      predef = augmentedPredef + System.lineSeparator() + predef,
```

Comment:
Could we provide an alias to `System.lineSeparator()` from something shorter? e.g. `newLine` or something in the Utils.scala